src/widgets/menu/defaultTemplates.tsx

```diff
@@ -7,17 +7,40 @@ import { cx } from '../../lib/utils';
 import type { MenuComponentTemplates } from './menu';
 
 const defaultTemplates: MenuComponentTemplates = {
-  item({ cssClasses, url, label, count }) {
+  item({
+    cssClasses,
+    url,
+    count,
+    highlighted,
+    label,
+    isFromSearch,
+    searchable,
+  }) {
     return (
       <a className={cx(cssClasses.link)} href={url}>
-        <span className={cx(cssClasses.label)}>{label}</span>
+        {searchable ? (
```

Comment:
note that this is different to the template for refinementList, to avoid a breaking change. In a major version it could be changed to always use `highlighted` (maybe this should be a comment)

Reply:
Instead of the big ternary, could we have the following in l.34?

```js
{!isFromSearch && searchable ? highlighted : label}
```

Reply:
that would use `label` when it's searchable, but not searching (so it would be inconsistent with refinement list). Another option however could be:

```
{searchable ? !isFromSearch && highlighted : label}
```

but that's essentially the same as the existing code, but with the ternary two times